Constants/TypeNames.cs

```diff
@@ -3,5 +3,8 @@ namespace Lombiq.Privacy.Constants
     public static class TypeNames
     {
         public const string ConsentCheckbox = nameof(ConsentCheckbox);
+        public const string ConsentBannerSettings = nameof(ConsentBannerSettings);
+        public const string ConsentCheckboxSettings = nameof(ConsentCheckboxSettings);
+        public const string RegistrationConsentSettings = nameof(RegistrationConsentSettings);
```

Comment:
Inkább legyenek "Privacy"-vel prefixelve.

Reply:
`ConsentCheckbox` is vagy csak ez a 3?

Reply:
Igazából akkor már az is, igen. Azért ez a három tűnt fel, mert a Configuration menüben is ott vannak.